src/dbnode/persist/fs/seek_manager.go

```diff
@@ -165,6 +180,8 @@ func NewSeekerManager(
 	return m
 }
 
+// Open opens a seeker, which essentially means opening the file descriptors
```

Comment:
This method actually opens the seeker manager which kicks off its background processes like the open/close loop.

Reply:
Changed it